Gems/GraphCanvas/Code/Source/Components/NodePropertyDisplays/VectorNodePropertyDisplay.cpp

```diff
@@ -355,14 +355,14 @@ namespace GraphCanvas
         {
             m_proxyWidget = new QGraphicsProxyWidget();
 
-            QWidget* widgetContainer  = new QWidget(nullptr);
-            QHBoxLayout* layout = new QHBoxLayout(widgetContainer);
-            widgetContainer->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::Fixed);
-            
+            m_widgetContainer = new QWidget();
+            QHBoxLayout* layout = new QHBoxLayout(m_widgetContainer);
+            m_widgetContainer->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::Fixed);
+
             layout->setAlignment(Qt::AlignLeft);
             layout->setContentsMargins(0, 0, 0, 0);
 
-            m_button = new QToolButton();
+            m_button = new QToolButton(m_widgetContainer);
```

Comment:
if it is a best practice to assign parent object while creating qt object, we should document it somewhere so others can follow the same standard to avoid this issue happens again

Reply:
Normally I do. This code was from graph canvas or script canvas. It was not the direct cause of the issues i was having. This update was a precaution.